common/djangoapps/util/views.py

```diff
@@ -76,7 +76,9 @@ def require_global_staff(func):
     """View decorator that requires that the user have global staff permissions. """
     @wraps(func)
     def wrapped(request, *args, **kwargs):  # pylint: disable=missing-docstring
-        if GlobalStaff().has_user(request.user):
+        course_id = kwargs.get('course_id')
+        course_key = CourseKey.from_string(course_id)
+        if GlobalStaff().has_user(request.user) or CourseInstructorRole(course_key).has_user(request.user):
```

Comment:
We need to provide this permission to course instructors as well as all course creators are instructors of the courses they created or are managing. So this modification would work for staff users and course instructors and in turn course creators.

Reply:
If edly cloud has roles like `course level staff` or `organization-level staff`, you will also need to handle that scenario as well.
![image](https://user-images.githubusercontent.com/42294172/88534020-abea2080-d020-11ea-8038-83cd504aa02e.png)

Reply:
we have an organization level staff but that is our custom made role. we only need to give permissions to anyone who is an instructor in that course. So, I think this will work. I am running build on dev server to test these nonetheless

Reply:
@rehan99000 we need to give course staff(`CourseStaffRole`) ability to see certs.

Reply:
okay I will add that as well